### PR TITLE
Editing pass

### PR DESCRIPTION
**Pull request by Buzzardo**
I edited for spelling, punctuation, grammar, usage, and corporate voice. I also corrected some links and made sure everything points uses https rather than http.

### CODE REVIEW

File: docs/pom.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-		 xmlns="http://maven.apache.org/POM/4.0.0"
-		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns:xsi="https://www.w3.org/2001/XMLSchema-instance"
```
Comment (pukkaone) [inline docs/pom.xml]: The namespace URI is treated only has an identifier. It is [not a goal](https://www.w3.org/TR/2009/REC-xml-names-20091208/#ns-decl) that it be directly usable for retrieval of a schema (if any exists). The URI should be exactly as documented. Changing the scheme to `https` actually makes it a different namespace name.

Comment (spencergibb) [reply docs/pom.xml]: Don't know that really matters here

Comment (Buzzardo) [reply docs/pom.xml]: The guidance I've gotten is to replace them. The build works either way.

**Comment by ryanjbaxter** [comment]
@Buzzardo mind looking at the conflicts?

File: docs/src/main/asciidoc/ribbon-integration.adoc [hunk 2]
```diff
@@ -1,40 +1,54 @@
 == Ribbon discovery in Kubernetes
 
 
-Spring Cloud client applications calling a microservice should be interested on relying on a client load-balancing
+Spring Cloud client applications that call a microservice should be interested on relying on a client load-balancing
 feature in order to automatically discover at which endpoint(s) it can reach a given service. This mechanism has been
-implemented within the [spring-cloud-kubernetes-ribbon](spring-cloud-kubernetes-ribbon/pom.xml) project where a
-Kubernetes client will populate a https://github.com/Netflix/ribbon[Ribbon] `ServerList` containing information
+implemented within the https://github.com/fabric8io/spring-cloud-kubernetes/tree/master/spring-cloud-kubernetes-ribbon[spring-cloud-kubernetes-ribbon] project, where a
```
Comment (pukkaone) [inline docs/src/main/asciidoc/ribbon-integration.adoc]: In the URL, the path segment `fabric8io` should be `spring-cloud`.

File: docs/src/main/asciidoc/discovery-client.adoc [hunk 3]
```diff
@@ -1,36 +1,58 @@
...
+This is something that you get for free by adding the following dependency inside your project:
 
-```java
+====
+[source,xml]
+----
+<dependency>
+    <groupId>org.springframework.cloud</groupId>
+    <artifactId>spring-cloud-starter-kubernetes</artifactId>
+    <version>${latest.version}</version>
```
Comment (pukkaone) [inline docs/src/main/asciidoc/discovery-client.adoc]: This reverses the change in issue https://github.com/spring-cloud/spring-cloud-kubernetes/issues/326 to remove version elements from the POM examples.

Comment (Buzzardo) [reply docs/src/main/asciidoc/discovery-client.adoc]: I didn't add that line, but I have removed it now.

File: docs/src/main/asciidoc/ribbon-integration.adoc (hunk 2 above)
Comment (Buzzardo) [reply docs/src/main/asciidoc/ribbon-integration.adoc]: Fixed. Thanks.

**Pull request merged**